### ISSUE TITLE
Allow copying Tangara serial port path/name

### ISSUE THREAD

**Issue by ticky** [haileys/tangara-companion#6]
Currently the companion displays but does not allow selecting or copying the serial port path. It’d be handy for developers to have it exposed as text or give it a copy button/click action.

**Comment by haileys** [comment]
Agreed, this is a good idea! Would you like to look into it? No pressure, I can take care of it if not

**Comment by ticky** [comment]
I’m not too familiar with Adwaita/GTK4 stuff, so I’m not sure what the right approach would be, I can take a dig though.

**Comment by ticky** [comment]
oh man I figured it out on my lunch break, you’re not going to believe how simple it is. will try to put up a PR tonight
